WordPressMocks.podspec

```diff
@@ -1,19 +1,21 @@
+# frozen_string_literal: true
+
 Pod::Spec.new do |s|
   s.name          = 'WordPressMocks'
-  s.version       = '0.0.12'
+  s.version       = '0.0.14'
```

Comment:
I see tag 0.0.13 exists still there's a skip there, is this because 0.0.13 was published but likely never merged into `trunk`?

Reply:
I assume that's the case, yeah. `0.0.13` was tagged just as I was ramping up as release conductor and I must have missed.